pkg/asset/installconfig/alibabacloud/validation.go

```diff
@@ -100,6 +121,92 @@ func validateResourceGroup(client *Client, ic *types.InstallConfig, path *field.
 	return append(allErrs, field.NotFound(path.Child("resourceGroupID"), ic.AlibabaCloud.ResourceGroupID))
 }
 
+func validateVpc(client *Client, ic *types.InstallConfig, path *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+	vpcs, err := client.ListVpcs(ic.AlibabaCloud.VpcID)
+	if err != nil {
+		return append(allErrs, field.InternalError(path.Child("vpcID"), err))
+	}
+	if vpcs.TotalCount == 0 {
+		allErrs = append(allErrs, field.NotFound(path.Child("vpcID"), ic.AlibabaCloud.VpcID))
+	}
+	return allErrs
+}
+
+func validateVSwitches(client *Client, ic *types.InstallConfig, path *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+
+	zoneIDs := []string{}
+	for idx, id := range ic.AlibabaCloud.VSwitchIDs {
+		vswitches, err := client.ListVSwitches(id)
+		if err != nil {
+			return append(allErrs, field.InternalError(path.Child("vswitchIDs"), err))
+		}
+		if vswitches.TotalCount != 1 {
+			allErrs = append(allErrs, field.NotFound(path.Child("vswitchIDs").Index(idx), id))
+			continue
+		}
+		if vswitches.VSwitches.VSwitch[0].VpcId != ic.AlibabaCloud.VpcID {
+			allErrs = append(allErrs, field.Invalid(path.Child("vswitchIDs").Index(idx), id, fmt.Sprintf("The VSwitch does not belong to vpc %s", ic.AlibabaCloud.VpcID)))
```

Comment:
All of the field error messages should being with lowercase letters.
```suggestion
 			allErrs = append(allErrs, field.Invalid(path.Child("vswitchIDs").Index(idx), id, fmt.Sprintf("the VSwitch does not belong to vpc %s", ic.AlibabaCloud.VpcID)))
```